sources/mongodb/__init__.py

```diff
@@ -71,6 +71,7 @@ def mongodb_collection(
     incremental: Optional[dlt.sources.incremental] = None,  # type: ignore[type-arg]
     write_disposition: Optional[str] = dlt.config.value,
     parallel: Optional[bool] = False,
+    limit: Optional[int] = None,
```

Comment:
Only adding into the resource, because `limit` in the source seems very questionable - there will be several collections, how to apply limit to them? To every collection? Or to all the collections together?